include/oneapi/dpl/internal/random_impl/linear_congruential_engine.h

```diff
@@ -172,6 +207,34 @@ class linear_congruential_engine
         }
     }
 
+    template <int _N = 0, bool _FLAG = false>
+    typename ::std::enable_if<(_N == 0) && (_FLAG == true)>::type
+    skip_seq(unsigned long long __num_to_skip)
+    {
+        ::std::uint64_t __mod = modulus, __inc = increment;
+        ::std::uint64_t __mult = pow_mult_n(__num_to_skip);
+        state_ = static_cast<scalar_type>((__mult * static_cast<::std::uint64_t>(state_) + __inc) % __mod);
+    }
+
+    template <int _N = 0, bool _FLAG = false>
+    typename ::std::enable_if<(_N == 1) && (_FLAG == true)>::type
+    skip_seq(unsigned long long __num_to_skip)
+    {
+        ::std::uint64_t __mod = modulus;
+        ::std::uint64_t __mult = pow_mult_n(__num_to_skip);
+        state_[0] = static_cast<scalar_type>((__mult * static_cast<::std::uint64_t>(state_[0])) % __mod);
+    }
+
+    template <int _N = 0, bool _FLAG = false>
+    typename ::std::enable_if<(_N > 1) && (_FLAG == true)>::type
+    skip_seq(unsigned long long __num_to_skip)
+    {
+        ::std::uint64_t __mod = modulus, __inc = increment;
+        ::std::uint64_t __mult = pow_mult_n(__num_to_skip);
+        state_ = ((__mult * state_.template convert<::std::uint64_t, cl::sycl::rounding_mode::rte>()) % __mod)
```

Comment:
minor: cl::sycl-> sycl::

Reply:
Fixed (also in uniform_integer_distribution)